lib/Game.js

```diff
@@ -4,7 +4,7 @@ class Game {
     this.currentRound = 1;
     this.puzzle = new Puzzle();
     this.currentPuzzle = this.puzzle.puzzleArray[this.currentRound-1];
-    this.currentWheel = null;
+    this.currentWheel = new Wheel();
   }
```

Comment:
Oh it makes sense to assign it to a new instance then to re-assign later within another function